.github/workflows/backstory-ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: find backstory/src/main -iname *.html | xargs node_modules/html-validate/bin/html-validate.js
     - name: Validate CSS
       if: always()
-      run: find backstory/src/main -iname *.css | xargs node_modules/css-validator/bin/css-validator
+      run: find backstory/src/main -iname *.css | xargs node_modules/csstree-validator/bin/validate
```

Comment:
For future reference, it would have been great if the validator changes could have been made in a separate PR. It better isolates the change (making it easier to find, revert, etc).

Reply:
I realized that after I made the commits. I agree it should have been its own PR. In the future, I'll make that a priority. Thank you!

Reply:
Done